gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -1067,6 +1067,9 @@ end
 -- Returns whether the player is ready. A player is ready when he is able to look
 -- around and move (first call of @{GM:SetupMove}). A bot player is always considered
 -- as ready.
+-- @note If called on the client, it is only 100% reliable for the local player. While the
+-- restult for other players is most likely still true, it can have some issues in certain
+-- scenarios.
```

Comment:
(optional)

I personally would bump this up to a warning as this is an unexpected caveat and might trip people up more than necessary.